index.html

```diff
@@ -0,0 +1,326 @@
+            </div>
+            <div class="work-details-row2">
+                <div class="wdr21">
+                    <p>See Project</p>
+                </div>
+                <div class="wdr22">
+                    <h4>Data Dashboard?Healthcare</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard</p>
+                    <div class="sp1"><span>HTML</span><span>CSS</span><span>Javascript</span></div>
+                </div>
+                <div class="wdr23">
+                    <h4>Website Protfolio</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard
+                    </p>
+                    <div class="sp1"><span>HTML</span><span>CSS</span><span>Javascript</span></div>
+                </div>
+            </div>
+            <div class="work-details-row3">
+                <div class="wdr31">
+                    <h4>Profesional Art Printing Data More</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard
+                    </p>
+                    <div class="sp1"><span>HTML</span><span>CSS</span><span>Javascript</span></div>
+                </div>
+
+                <div class="wdr32">
+                    <h4>Data Dashboard?Healthcare</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard
+                    </p>
+                    <div class="sp1"><span>HTML</span><span>CSS</span><span>Javascript</span></div>
+                </div>
+                <div class="wdr33">
+                    <h4>Website Protfolio</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard
+                    </p>
+                    <div class="sp1"><span>HTML</span><span>CSS</span><span>Javascript</span></div>
+                </div>
+            </div>
+
+
+        </div>
+
+    </div> --->
+    <!------------------about me------------------------------------------------>
+    <section>
+        <div class="about-me">
+            <div class="about-me-row1">
+                <ul class="about-me-row1-left">
+                    <li class="dashboard">About me</li>
+                    <li class="description-about-me-row1-left">Hey there! I can help you build a product, feature or website. Look through some of my work and
+                        experience! If you like
+                        what you see and have a project you need coded, don’t hesitate to contact me.
+                    </li>
+                    <li class="seeprojects1">Get My Resume</li>
```

Comment:
- [ ] Kindly make sure this background image is placed in the section, so it sticks to the Figma design

![Screenshot from 2023-01-19 21-59-39](https://user-images.githubusercontent.com/80458199/213547787-763f3042-906d-4a2a-b0ff-3dbbdcc9f622.png)